frontend/src/components/Notify.tsx

```diff
@@ -28,6 +39,74 @@ export const Notify = () => {
       setMessage("");
     };
 
+    const fetchTestById = async (id: string) => {
+      console.log(id, "im id");
+      try {
+        let url = `http://localhost:50000/v1.0/invoke/catalogmanager/method/test/${id}`;
```

Comment:
URLs must come from a central configuration. They are going to be different once we deploy 
the backend in the cloud (soon)